synapse/config/server.py

```diff
@@ -993,6 +996,14 @@ def generate_config_section(
         # act as if no error happened and return a fake session ID ('sid') to clients.
         #
         #request_token_inhibit_3pid_errors: true
+
+        # Number of forward extremities in a room at which Synapse will send a dummy
+        # events.
+        # This is to prevent forward extremities from building up in the room, which
+        # could lead to poor performance.
```

Comment:
"a dummy events"

Generally this could do with a bit of rewording; it puts the horse before the cart. 

> Forward extremities can build up in a room due to networking delays between
> homeservers. Once this happens in a large room, calculation of the state of
> that room can become quite expensive.  To mitigate this, once the number of
> forward extremities reaches a given threshold, Synapse will send an 
> `m.room.dummy` [or whatever] event, which will reduce the forward extremities 
> in the room.
> 
> This setting defines the threshold at which dummy events are sent. The defualt is 10.

Reply:
also, it seems to be in a section called `## Homeserver blocking ##`. Please could you put it elsewhere (maybe in the miscellaneous section at the top).

Reply:
> also, it seems to be in a section called `## Homeserver blocking ##`. Please could you put it elsewhere (maybe in the miscellaneous section at the top).

Ugh, I hadn't noticed that. Though a lot of settings in there (retention, redaction censoring, how long to track user IPs, etc) have nothing to do with blocking. I'll move it upwards anyway.